kasa/aestransport.py

```diff
@@ -35,6 +35,10 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+ONE_DAY_SECONDS = 86400
+SESSION_EXPIRE_BUFFER_SECONDS = 60 * 20
```

Comment:
I didn't put them in a separate const file because there is no guarantee that aes and klap will always have the same session length

Reply:
The device reports the timeout in the response headers, so if we start getting reports about unavailable devices on regular intervals, we may look into reading the value from there.

Reply:
That would be better for sure. I think we should get if from there and than subtract the buffer for clock sync

Reply:
Yup. It's in the cookie header: `CIMultiDictProxy('Set-Cookie': 'TP_SESSIONID=xx;TIMEOUT=86400', 'Server': 'SHIP 2.0', 'Content-Length': '48', 'Content-Type': 'text/html')>`

Reply:
Looks like its the same for AES and KLAP